src/ResourceManager/RecoveryServices.Backup/RecoveryServices.Backup.sln

```diff
@@ -21,7 +23,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{54815A13
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.RecoveryServices.Backup.Test", "Commands.RecoveryServices.Backup.Test\Commands.RecoveryServices.Backup.Test.csproj", "{65E15D74-1E24-4600-8671-D0B420524B17}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.ScenarioTests.ResourceManager.Common", "..\Common\Commands.ScenarioTests.ResourceManager.Common\Commands.ScenarioTests.ResourceManager.Common.csproj", "{3436A126-EDC9-4060-8952-9A1BE34CDD95}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.RecoveryServices.Test", "..\RecoveryServices\Commands.RecoveryServices.Test\Commands.RecoveryServices.Test.csproj", "{6C7D3D81-37AB-445E-8081-78A1FEC0A773}"
```

Comment:
Is this the common RecoveryServices test project? If yes, then why we need it in RecoveryServices.Backup sln?